server/src/controllers/tokens.ts

```diff
@@ -213,17 +219,17 @@ export class TokensController {
     @QueryParam('pool') pool: string,
     @QueryParam('key') key: string,
   ): Promise<TokenBalance[]> {
-    const poolMap = new Map<string, string>();
+    const poolMap = new Map<string, TokenPool>();
     const balances = await firefly.getTokenBalances({ pool, key, balance: '>0' });
     for (const b of balances) {
       if (!poolMap.has(b.pool)) {
         const pool = await firefly.getTokenPool(b.pool);
-        poolMap.set(b.pool, pool.name);
+        poolMap.set(b.pool, pool);
       }
     }
     return balances.map((b) => ({
       pool: b.pool,
-      poolName: poolMap.get(b.pool),
+      poolObject: poolMap.get(b.pool),
```

Comment:
Maybe we should just expose the pool object as `pool`, instead of having two different keys here.